eZ/Bundle/EzPublishCoreBundle/Composer/ScriptHandler.php

```diff
@@ -119,24 +119,18 @@ public static function installWelcomeText(Event $event)
 <fg=cyan>Welcome to eZ Platform!</fg=cyan>
 
 <options=bold>Quick Install:</>
-(Assuming the CLI user you execute commands with below is same that extracted/installed the software)
-<comment>    $  export SYMFONY_ENV="prod"</comment>
+<comment>    $  export SYMFONY_ENV="dev"</comment>
 <comment>    $  php ${consoleDir}/console ezplatform:install <type></comment>
 <comment>    $  php ${consoleDir}/console assetic:dump</comment>
 <comment>    $  php ${consoleDir}/console server:run</comment>
 
 Note:
 - "ezplatform:install" has different installer <type>s depending on your install, see <fg=green>INSTALL.md</> or <fg=green>README.md</> for which one to use.
-- For development use you can enable full debugging by setting SYMFONY_ENV to "dev".
-- Last command will give you url to frontend of installation, add "/ez" to reach backend.
+- Instructions above assumes the CLI user you execute commands with is same that extracted/installed the software.
```

Comment:
Hmm, something here doesn't sound right for me. Maybe:
`The instructions above assume the CLI user you execute commands with is the same that extracted/installed the software.`
Not an expert, I think it's safer to ping @DominikaK :grinning:

Reply:
wdyt @DominikaK ? :)

Reply:
Agree with @alongosz, maybe even `is the same one that`...

Reply:
@DominikaK maybe add `these` to? `The instructions above assume the CLI user you execute these commands with is the same one that extracted/installed the software.` ?

Reply:
Even better :)

Reply:
Fixed in 492847b